nixos/modules/virtualisation/vmware-guest.nix

```diff
@@ -25,6 +25,8 @@ in
         serviceConfig.ExecStart = "${open-vm-tools}/bin/vmtoolsd";
       };
 
+    environment.etc."vmware-tools".source = "${pkgs.open-vm-tools}/etc/vmware-tools/*";
```

Comment:
This commit fixes an error message on the host VMware

> The VMware Tools resume script did not run successfully in this virtual machine

> The VMware Tools power-on script did not run successfully in this virtual machine. If you have configured a custom power-on script in this virtual machine, make sure that it contains no errors. You can also submit a support request to report this issue.

I'm not sure if those scripts are really necessary, but getting an warning at startup and suspend/resume was annoying.